File: Java/1828. Queries on Number of Points Inside a Circle.java
class Solution {
    public int[] countPoints(int[][] points, int[][] queries) {
        List<Integer> ans = new ArrayList<Integer>();
        int temp;

        for(int i = 0; i < queries.length; i++){
            temp = 0;
            for(int j = 0; j < points.length; j++){
                if(Math.pow(queries[i][0] - points[j][0], 2) + Math.pow(queries[i][1] - points[j][1], 2) <= Math.pow(queries[i][2], 2)) temp += 1;
            }
            ans.add(temp);
        }

        int res[] = new int[ans.size()];
        for(int i = 0; i < ans.size(); i++) res[i] = (int)ans.get(i);
        return res;
    }
}

File: Python/1828. Queries on Number of Points Inside a Circle.py
class Solution:
    def countPoints(self, points: List[List[int]], queries: List[List[int]]) -> List[int]:
        res = []

        for x, y, r in queries:
            temp = 0
            for i, j in points:
                if (x - i) ** 2 + (y - j) ** 2 <= r ** 2: temp += 1
            res.append(temp)

        return res
